### PR TITLE
Electron-318 (Included build number in the client version info)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
## Description
Included the build number in the version info API [ELECTRON-318](https://perzoinc.atlassian.net/browse/ELECTRON-318)

## Approach
How does this change address the problem?
- #### Problem with the code: Build number was missing in the `getVersionInfo` method
- #### Fix: Fetching build number from `package.json` and added to the `getVersionInfo`

### Spectron test results
- [Electron-318 — Spectron.pdf](https://github.com/symphonyoss/SymphonyElectron/files/1802259/Electron-318.Spectron.pdf)

### Unit test results
- [Electron-318 — Unit Tests.pdf](https://github.com/symphonyoss/SymphonyElectron/files/1802260/Electron-318.Unit.Tests.pdf)

## Open Questions if any and Todos
- [x] Unit-Tests
- [x] Documentation
- [x] Automation-Tests
When solved, check the box and explain the answer.

@VikasShashidhar & @VishwasShashidhar Please review. Thanks 😄

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -219,14 +243,19 @@ <h1>Symphony Electron API Demo</h1>
           console.log(e)
         }
 
-        var getSources = document.getElementById('get-version');
-        getSources.addEventListener('click', function() {
+        let getVersionInfo = document.getElementById('get-version');
```
[the remark this answers is not in the document]
Comment (KiranNiranjan) [reply demo/index.html]: This code only runs on `Electron`. but I have changed it to var to make it consistent across the file.

**Comment by KiranNiranjan** [comment]
@NitinSingh2020 👍

**Comment by VishwasShashidhar** [comment]
lgtm 👍

**Comment by KiranNiranjan** [comment]
@NitinSingh2020 It is preserved 👍 I just added `Squash` `...`

**Comment by VishwasShashidhar** [comment]
@KiranNiranjan Please add the unit & spectron tests results for this branch.

**Pull request merged**